clients/command-amqp/src/main/java/org/eclipse/hono/client/command/amqp/ProtonBasedCommandContext.java

```diff
@@ -70,38 +74,56 @@ public ProtonBasedCommand getCommand() {
 
     @Override
     public void accept() {
+        Tags.HTTP_STATUS.set(getTracingSpan(), HttpURLConnection.HTTP_OK);
         updateDelivery(Accepted.getInstance());
     }
 
     @Override
     public void release() {
+        TracingHelper.logError(getTracingSpan(), "command could not be delivered or processed");
+        Tags.HTTP_STATUS.set(getTracingSpan(), HttpURLConnection.HTTP_UNAVAILABLE);
         updateDelivery(Released.getInstance());
     }
 
     @Override
     public void release(final Throwable error) {
         Objects.requireNonNull(error);
+        TracingHelper.logError(getTracingSpan(), "command could not be delivered or processed", error);
+        final int status = error instanceof ServerErrorException
+                ? ((ServerErrorException) error).getErrorCode()
+                : HttpURLConnection.HTTP_INTERNAL_ERROR;
+        Tags.HTTP_STATUS.set(getTracingSpan(), status);
         updateDelivery(Released.getInstance());
     }
 
     @Override
     public void modify(final boolean deliveryFailed, final boolean undeliverableHere) {
+        final int status = undeliverableHere ? HttpURLConnection.HTTP_NOT_FOUND
+                : HttpURLConnection.HTTP_UNAVAILABLE;
+        Tags.HTTP_STATUS.set(getTracingSpan(), status);
```

Comment:
is there a particular reason why we do not log an error to the Span here?

Reply:
I've added log output.